ui/src/app/subnets/views/VLANDetails/VLANDetails.tsx

```diff
@@ -63,10 +65,16 @@ const VLANDetails = (): JSX.Element => {
 
   return (
     <Section header={<VLANDetailsHeader id={id} />}>
-      <VLANSummary id={id} />
-      <DHCPStatus id={id} />
-      <ReservedRanges />
-      <VLANSubnets id={id} />
+      {showDHCPForm ? (
+        <ConfigureDHCP closeForm={() => setShowDHCPForm(false)} id={id} />
+      ) : (
+        <>
+          <VLANSummary id={id} />
+          <DHCPStatus id={id} openForm={() => setShowDHCPForm(true)} />
+          <ReservedRanges />
+          <VLANSubnets id={id} />
+        </>
+      )}
       <DHCPSnippets
```

Comment:
I think this should hide the DHCPSnippets as well.

Reply:
@huwshimi I copied the functionality from legacy, assuming this info could be useful for someone configuring DHCP. What do you think?

Reply:
Oh yeah I see now. It's there when you enable, but not when you reconfigure. I'm not totally sure it's meant to be there, but let's leave it there for now.

Reply:
hahaha of course it would be inconsistent in legacy